spec/workers/downstream_draft_worker_spec.rb

```diff
@@ -1,4 +1,5 @@
 require "rails_helper"
+require "sidekiq_unique_jobs/testing"
```

Comment:
It'd be more conventional to put this amongst the other requires in the rails helper: https://github.com/alphagov/publishing-api/blob/74334bd9785495c1443db8d849fc7dbfedc785cb/spec/rails_helper.rb#L15